app/src/main/java/com/example/next_show/MainActivity.java

```diff
@@ -94,43 +96,43 @@ private void setProfileFragment(ProfileFragment profileFragment) {
     }
 
     private void grabTraktData() {
-        Thread thread = new Thread(new Runnable() {
-            @Override
-            public void run() {
-                try  {
-                    // Trakt wrapper
-                    TraktV2 trakt = new TraktV2(getString(R.string.trakt_client_id));
-                    Shows traktShows = trakt.shows();
-                    try {
-                        // Get trending shows
-                        Response<List<TrendingShow>> response = traktShows.trending(1, null, Extended.FULL).execute();
-                        if (response.isSuccessful()) {
-                            List<TrendingShow> repsonseShows = response.body();
-                            for (TrendingShow trending : repsonseShows) {
-                                Log.i(TAG, "Title: " + trending.show.title);
-                                Show currentShow = new Show(trending.show.title);
-                                shows.add(currentShow);
-                            }
-
-                            // show the list
-                            Log.i(TAG, "List of show titles: " + shows.toString());
+        // Trakt wrapper
+        TraktV2 trakt = new TraktV2(getString(R.string.trakt_client_id));
+        Shows traktShows = trakt.shows();
+
+        try {
+            traktShows.trending(1, null, Extended.FULL).enqueue(new Callback<List<TrendingShow>>() {
```

Comment:
Unfortunately, Java doesn't have named parameters.  So, that means that 1 is a magic number and it's unclear what null means.

For 1, let's save this into a static constant with better naming.  For null, we can add a comment next to that says what it is, such as `null /* should poll for requests */`